gradle/publish.gradle

```diff
@@ -60,55 +64,10 @@ bintray {
 
 def isMultiplatform = project.name == "napier"
 
-afterEvaluate {
-    task sourcesJar(type: Jar) {
-        classifier = 'sources'
-        if (isMultiplatform) {
-            from kotlin.sourceSets.commonMain.kotlin
-        } else {
-            from sourceSets.main.allSource
-        }
-    }
-}
-
 task javadocJar(type: Jar) {
     archiveClassifier = 'javadoc'
 }
 
-afterEvaluate {
-    publishing {
-        def projectName = project.name
-
-        publications.all {
-            artifactId projectName
-            pom.withXml {
-                def root = asNode()
-                root.appendNode('name', projectName)
-                root.appendNode('description', POM_DESCRIPTION)
-                root.appendNode('url', SITE_URL)
-                root.children().last() + pomConfig
-            }
-
-            // rename artifacts
-            groupId = BINTRAY_PACKAGE
-            if (name == 'kotlinMultiplatform') {
```

Comment:
I think we still need these lines.
I know we write dependencies only one time in a common module on Kotlin 1.4.

Kotlin Coroutines has module files in core artifact.
https://repo1.maven.org/maven2/org/jetbrains/kotlinx/kotlinx-coroutines-core/1.3.9/
But I can't make this file on Kotlin 1.4.
Do you have any idea?

Reply:
This bug is caused by bintray plugin.

 https://github.com/bintray/gradle-bintray-plugin/issues/229